app/src/main/java/fr/neamar/kiss/result/DuckduckgoSearchResult.java

```diff
@@ -0,0 +1,47 @@
+package fr.neamar.kiss.result;
+
+import android.annotation.TargetApi;
+import android.app.SearchManager;
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.view.View;
+import android.widget.TextView;
+
+import fr.neamar.kiss.R;
+import fr.neamar.kiss.pojo.DuckduckgoSearchPojo;
+
+public class DuckduckgoSearchResult extends Result {
+    private final DuckduckgoSearchPojo searchPojo;
+
+    public DuckduckgoSearchResult(DuckduckgoSearchPojo searchPojo) {
+        super();
+        this.pojo = this.searchPojo = searchPojo;
+    }
+
+    @Override
+    public View display(Context context, int position, View v) {
+        if (v == null)
+            v = inflateFromId(context, R.layout.item_search);
+
+        TextView appName = (TextView) v.findViewById(R.id.item_search_text);
+        appName.setText(enrichText("DuckDuckGo \"{" + searchPojo.query + "}\""));
+
+        return v;
+    }
+
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
```

Comment:
is there anything requiring Honeycomb?

The app is targeted for Android < 3.0 too